app/models/concerns/labware_creators/donor_pooling_calculator.rb

```diff
@@ -226,14 +253,14 @@ def adjust_number_of_cells_per_chip_well(count_of_samples_in_pool, number_of_cel
     # calculate chip loading volume
     chip_loading_volume = calculate_chip_loading_volume(number_of_cells_per_chip_well)
 
-    # calculate full allowance
-    full_allowance = calculate_full_allowance(chip_loading_volume)
+    # calculate allowance
+    allowance = calculate_allowance(chip_loading_volume, allowance_band)
 
     # do not adjust existing value if we have enough volume
-    return number_of_cells_per_chip_well if final_suspension_volume >= full_allowance
+    return number_of_cells_per_chip_well if final_suspension_volume >= allowance
 
     # we need to adjust the number of cells per chip well according to the number of samples
-    Rails.application.config.scrna_config[:full_allowance_table][count_of_samples_in_pool]
+    Rails.application.config.scrna_config[:allowance_table][allowance_band][count_of_samples_in_pool]
```

Comment:
I think we should put something in to check this hasn't returned `nil` - I guess it should error out if it does. It shouldn't hit those values because, for instance, if you have 8 samples, you should always have enough material for '2 pool attempts, 1 count'.

Reply:
Yes I wasn't sure if this needed specific handling. I suspect it will error later down the line but a descriptive error would be useful here.

Reply:
Added error handling in 
https://github.com/sanger/limber/pull/2212/commits/2279f54606c333c75bd71c2511f50b453ebeac57